beacon-chain/blockchain/log.go

```diff
@@ -21,7 +22,17 @@ func logStateTransitionData(b *ethpb.BeaconBlock) {
 		"attesterSlashings": len(b.Body.AttesterSlashings),
 		"proposerSlashings": len(b.Body.ProposerSlashings),
 		"voluntaryExits":    len(b.Body.VoluntaryExits),
-	}).Info("Finished applying state transition")
+	}).Debug("Finished applying state transition")
+}
+
+func logSyncStatus(block *ethpb.BeaconBlock, blockRoot [32]byte, finalized *ethpb.Checkpoint) {
+	log.WithFields(logrus.Fields{
+		"slot":           block.Slot,
+		"block":          fmt.Sprintf("0x%s...", hex.EncodeToString(blockRoot[:])[:8]),
+		"epoch":          helpers.SlotToEpoch(block.Slot),
+		"finalizedEpoch": finalized.Epoch,
```

Comment:
My argument for this is 99% of users/non developers don't care about `finalizedEpoch` and `finalizedRoot` when syncing. So perhaps move it to debug?

Reply:
Not seeing the finality is my main pain point. Everyone should care about finality :)

Burying it in `DEBUG` level would be very unfortunate due to the noise.

Reply:
Marinated a bit more with this. I'm on board with this rationale